include/prism/node.h

```diff
@@ -33,6 +33,7 @@ PRISM_EXPORTED_FUNCTION const char * pm_node_type_to_str(pm_node_type_t node_typ
 // declare them here to avoid generating them.
 typedef struct pm_scope_node {
     pm_node_t base;
+    pm_node_t parent;
```

Comment:
Needs *

Reply:
Thanks for this. And my apologies! I pushed this up in a rush before a meeting